CONTRIBUTING.md

```diff
@@ -23,3 +23,12 @@ contributors in the pull request's body or in subsequent comments.
 
  If you are making a pull request on behalf of someone else but you had no part in designing the 
  feature, you can remove yourself with the above syntax.
+
+# Tests
+
+For normative changes, a corresponding
+[web-platform-tests](https://github.com/w3c/web-platform-tests) PR is highly appreciated. Typically,
+both PRs will be merged at the same time. Note that a test change that contradicts the spec should
+not be merged before the corresponding spec change. If testing is not practical, please explain why
+and if appropriate [file an issue](https://github.com/w3c/web-platform-tests/issues/new) to follow
+up later. Add the `type:untestable` or `type:missing-coverage` label as appropriate.
```

Comment:
wrt this sentence "If testing is not practical..." -- is seems to be in reference to a presumed normative-but-not-practically-testable webauthn PR, yes?

wrt the "Add the `type:untestable` or `type:missing-coverage` label..."  sentence -- add the label to the presumed normative-but-not-practically-testable webauthn PR or to the newly-submitted web-platform-tests issue?